descriptors/ModuleDescriptor-template.json

```diff
@@ -116,6 +116,38 @@
         }
       ]
     },
+    {
+      "id": "custom-fields",
+      "version": "1.0",
+      "interfaceType" : "multiple",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/custom-fields",
+          "permissionsRequired": ["custom.fields.collection.get"]
```

Comment:
Please rename all permissions to start with "user-settings.", like "user-settings.custom.fields.collection.get" instead of "custom.fields.collection.get".

For other modules we would need differently named permissions so we cannot use simply "custom.fields.collection.get" w/o a prefix.

Reply:
thanks, @dmtkachenko, renamed